config/config.go

```diff
@@ -46,6 +46,10 @@ const (
 	SSLKeyPathKey = "SSL_KEY"
 	// MnemonicKey is the mnemonic of the master private key of the daemon's wallet
 	MnemonicKey = "MNEMONIC"
+	// Enables profiler that can be used to investigate performance issues
+	EnableProfiler = "ENABLE_PROFILER"
+	// ProfilerPort defines profiler port
+	ProfilerPort = "PROFILER_PORT"
```

Comment:
I think for sake of simplicity we just decided an hardcoded port and be with it 

```suggestion
```

Reply:
Done